controllers/account.controller.ts

```diff
@@ -0,0 +1,65 @@
+import { Request, Response } from "express";
+import { AccountService } from "../services/account.service";
+import { errorResponse, notFound, serverError, successResponse } from '../util/response.helper';
+import { ResponseMessage } from "../contants/response-message.contant";
+import { IAccount } from "../models/interface/account.model.interface";
+import { comparePassword, createToken } from "../util/helper";
+
+export class AccountController {
+
+    private accountService: AccountService;
+
+    constructor() {
+        this.accountService = new AccountService();
+    }
+
+    public async registerAccount(request: Request, response: Response) {
+        try {
+
+            const account: IAccount = await this.accountService.registerAccount(request.body);

```

Comment:
you can remove `IAccount` type from `const account` variable.

Reply:
removed